### ISSUE TITLE
Try catch issue

### ISSUE THREAD

**Issue by balaji8385** [eslint/eslint#11701]
**Tell us about your environment**

* **ESLint Version:** 5.11.1
* **Node Version:** 8.10.0
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

```

**What did you expect to happen?**
Should have notified nested try catch blocks.

**What actually happened? Please include the actual, raw output from ESLint.**
No error was notified

**Are you willing to submit a pull request to fix this bug?**
No
